sbi/utils/posterior_ensemble.py

```diff
@@ -0,0 +1,446 @@
+import torch
+from sbi.utils import gradient_ascent, mcmc_transform
+from sbi.utils.torchutils import ensure_theta_batched
+from sbi.utils.user_input_checks import process_x
+from sbi.inference.posteriors.base_posterior import NeuralPosterior
+from sbi.inference.potentials.base_potential import BasePotential
+
+from torch import Tensor
+from typing import List, Optional, Union, Any
+from sbi.types import Shape, TorchTransform
+
+
+class NeuralPosteriorEnsemble(NeuralPosterior):
+    r"""Wrapper class for bundling together different posterior instances into an
```

Comment:
The first line of a docstring should fit into a single line. I suggest removing the word `class`